### PR TITLE
Polls 2020 08 03

### PR DESCRIPTION
**Pull request by LongForWisdom**
Polls for the week beginning 03/08/20

### CODE REVIEW

File: governance/polls/Base Rate Adjustment - August 03, 2020.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+   6: 0
+   7: 0.25
+   8: 0.5
+   9: 1
+   10: 2
+   11: 4
+---
+# Poll: Base Rate Adjustment - August 03, 2020
+
+The Governance Facilitators have placed a governance poll into the [voting system](https://vote.makerdao.com/polling) which presents several possible **Base Rate** options. Voters are now able to signal their support for a **Base Rate** within a range of-4% to 4%
```
Comment (ultraschuppi) [inline governance/polls/Base Rate Adjustment - August 03, 2020.md:22]: as I guess this a very common misunderstanding about the negative baserate, i would put it as topmost as possible and as bold, blinking and maybe even use `<marquee>` for it stating that a negative base rate is *not* leading to a negative stability for any asset as there is a lower boundary of 0 for every stability fee on any collateral

i am pretty bad with words, so no real suggestion from my side ;-)

Comment (hexonaut) [reply governance/polls/Base Rate Adjustment - August 03, 2020.md:22]: Agree with this. Here is some text:

Stability Fees are calculated from the Base Rate and Risk Premiums using the formula `Stability Fee = max(Base Rate + Risk Premium, 0%)`. Please note that a negative **Base Rate** does **NOT** imply negative **Stability Fees**.

Comment (LongForWisdom) [reply governance/polls/Base Rate Adjustment - August 03, 2020.md:22]: Great, thanks guys, I'll add this in tomorrow.

**Comment by LongForWisdom** [comment]
Just updated taking into account feedback. Thanks to both @hexonaut and @ultraschuppi

**Pull request merged**